src/components/FlowerCard.tsx

```diff
@@ -0,0 +1,33 @@
+import { FC } from "react";
+import Post from "../models/postModel";
+
+interface FlowerProps {
+  props: Post;
+}
+
+const FlowerCard: FC<FlowerProps> = (props) => {
```

Comment:
instead = (props ) => you can do = ({ id, name, latin_name, sightings, profile_picture }) =>